File: python/czas.py
czas = int(input())

sekundy = czas % 60
minuty = int((czas - sekundy) / 60)
godziny = int(minuty / 60)
minuty = int(minuty - godziny*60)

print(f"{godziny}g{minuty}m{sekundy}s")

File: c++/czas.cpp
#include <iostream>

using namespace std;

int main(){
    int czas;

    cin >> czas;

    int sekundy = czas % 60;
    int minuty = (czas - sekundy) / 60;
    int godziny = minuty / 60;
    minuty = minuty - godziny*60;
    
    cout << godziny << "g" << minuty << "m" << sekundy << "s";
}